blueocean-dashboard/src/main/js/components/LogConsole.jsx

```diff
@@ -1,7 +1,5 @@
 import React, { Component, PropTypes } from 'react';
-import { scrollToHash } from './ScrollToHash';
-
-const { array, bool, string } = PropTypes;
+import { scrollHelper } from './ScrollHelper';
 
 const INITIAL_RENDER_CHUNK_SIZE = 100;
 const INITIAL_RENDER_DELAY = 300;
```

Comment:
@michaelneale yes, that is coming from @cliffmeyers when he implemented the "buffered rendering". @sophistifunk I have no better idea ATM how to do the "increment rendering" so did not touched it, only adopted the code to fix the bugs it causes.